### ISSUE TITLE
[Can't connect instance because ipv6]

### ISSUE THREAD

**Issue by letanloc1998** [canonical/multipass#1934]
**Describe the bug**
I can't connect my instance via multipass. I access instance via Hyper-V => eth0 using ipv6 (inet => inet6)

![image](https://user-images.githubusercontent.com/26348718/105653924-8666a280-5eef-11eb-8d1f-c1c70f6506f5.png)

**To Reproduce**

**Expected behavior**

**Logs**

**Additional info**
 - OS: Windows

**Additional context**

**Comment by ricab** [comment]
Hi @letanloc1998, did you disable the IPv4 manually? Multipass needs it to connect to the instance.

**Comment by letanloc1998** [comment]
> Hi @letanloc1998, did you disable the IPv4 manually? Multipass needs it to connect to the instance.

No, I didn't disable IP4. My instance lost IP4, I access into HyperV to fix this error
